### PR TITLE
Add termshark 2.3.0

### PR DESCRIPTION
**Pull request by Slach**
fixes from https://github.com/ScoopInstaller/Main/pull/3132#issuecomment-1002111786
and from https://github.com/ScoopInstaller/Extras/pull/5420 applied

### CODE REVIEW

File: bucket/termshark.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+    "version": "2.3.0",
+    "description": "A terminal UI for tshark, inspired by Wireshark",
+    "homepage": "https://github.com/gcla/termshark",
+    "license": "MIT",
+    "depends": "wireshark",
+    "notes": [
+        "Run 'scoop install wireshark' (from a elevated shell) to install 'tshark.exe' automatically"
+    ],
+
```
Comment (rashil2000) [inline bucket/termshark.json:10]:
```suggestion
    "depends": "extras/wireshark",
```

File: bucket/termshark.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            "hash": "6c59c1d4f856f5611242588826274a93422a9ca9052b0a0c15a13177a604fcf9",
+            "extract_dir": "termshark_2.3.0_windows_x64"
+        }
+    },
+    "shortcuts": [
+        [
+            "termshark.exe",
+            "termshark"
+        ]
+    ],
```
Comment (rashil2000) [inline bucket/termshark.json:23]:
```suggestion
    "bin": "termshark.exe",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/termshark.json (hunk 1 above)
Comment (Slach) [reply bucket/termshark.json:10]: @rashil2000 `tshark` is optional requirements, and need only for packet capture
also we have multiple `wireshark` package variants `wireshark-np` for example

Comment (rashil2000) [reply bucket/termshark.json:10]: > @rashil2000 `tshark` is optional requirements, and need only for packet capture 

How can it be optional if it depends on it? It doesn't work without tshark. See: 

image


> also we have multiple `wireshark` package variants `wireshark-np` for example

The extras bucket only has 'wireshark'

**Comment by Slach** [comment]
@rashil2000 all changes done

**Pull request merged**